apps/rehash.c

```diff
@@ -267,6 +267,21 @@ static void str_free(char *s)
     OPENSSL_free(s);
 }
 
+static int slashy_dir_end_needed(const char *path)
```

Comment:
I'm not keen on those immediate returns, as both the Windows and VMS C libraries can handle Unix-style paths as well.  But other than that, sure.  Coming up.